src/Timers.cpp

```diff
@@ -712,7 +712,7 @@ PVR_ERROR Timers::AddTimer(const kodi::addon::PVRTimer& timer)
 
   case TIMER_REPEATING_EPG:
     if (timer.GetClientChannelUid() == PVR_TIMER_ANY_CHANNEL)
-    // Fake a manual recording not a specific type in NextPVR
+      // Fake a manual recording not a specific type in NextPVR
```

Comment:
Move the comment inside the brace if that is what it pertains to. Otherwise, before. 

The two space indent is confusing here.

Reply:
@phunkyfish thanks, totally agree, and I re-structured all the comments to make them less confusing.